collector/cmd/otelarrowcol/build.yaml

```diff
@@ -0,0 +1,47 @@
+dist:
+  module: github.com/open-telemetry/otel-arrow/collector/cmd/otelarrowcol
+  name: otelarrowcol
+  description: Development OTel-Arrow Collector binary, testing only.
+
+exporters:
+  - import: go.opentelemetry.io/collector/exporter/loggingexporter
+    gomod: go.opentelemetry.io/collector/exporter/loggingexporter v0.83.0
+  - import: go.opentelemetry.io/collector/exporter/otlphttpexporter
+    gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.83.0
+  - import: github.com/open-telemetry/otel-arrow/collector/exporter/otelarrowexporter
+    gomod: github.com/open-telemetry/otel-arrow/collector v0.0.0
+  - import: github.com/open-telemetry/otel-arrow/collector/exporter/fileexporter
+    gomod: github.com/open-telemetry/otel-arrow/collector v0.0.0
+
+receivers:
+  - import: github.com/open-telemetry/otel-arrow/collector/receiver/otelarrowreceiver
+    gomod: github.com/open-telemetry/otel-arrow/collector v0.0.0
+  - import: github.com/open-telemetry/otel-arrow/collector/receiver/filereceiver
+    gomod: github.com/open-telemetry/otel-arrow/collector v0.0.0
+  # TODO note the t-g repo needs a release, have used the latest sha below
```

Comment:
Can this release be done before this change merged? I'm not seeing the sha below

Reply:
Yeah I realize that builder doesn't like having a SHA in this position so I released it. Probably no big deal but there was a bug and a fix in that repo, so :+1: